test_project/store/src/main/java/test/sergeyvalyushko/store/helpers/RandomStorePopulator.java

```diff
@@ -0,0 +1,19 @@
+package test.sergeyvalyushko.store.helpers;
+
+import com.github.javafaker.Faker;
+import test.sergeyvalyushko.store.Product;
+
+import java.util.Date;
+import java.util.List;
+
+public class RandomStorePopulator {
+    Faker faker = new Faker();
+    public List<Product> createData(String categoryName, List<Product> productList) {
+        switch (categoryName){
+            case ("Book"): productList.add(new Product(faker.book().title(), (int) ((Math.random() * 10)+1), faker.date().between(new Date(1212121212222L), new Date()))); break;
```

Comment:
it is better to delegate the responsibility of populating product fully to populator, and if you invoke it on category is should do all the job including the number of products and not duplicate this code in other places